services/resource-usage-tracker/tests/unit/modules/prometheus_containers/test_resource_tracker.py

```diff
@@ -32,16 +32,20 @@ def app_environment(
 @pytest.fixture
 def mock_background_task(mocker: MockerFixture) -> mock.Mock:
     mocked_task = mocker.patch(
-        "simcore_service_resource_usage_tracker.resource_tracker.collect_container_resource_usage_task",
+        "simcore_service_resource_usage_tracker.modules.prometheus_containers.plugin.collect_container_resource_usage_task",
         autospec=True,
     )
     return mocked_task
 
 
+@pytest.mark.skip(
+    reason="This test is currently not needed, as setup_background_task is commented out in application.py"
```

Comment:
is there any point in keeping it in the code? or do you plan to add it again? if not just remove dead code while you still know it is to be removed.

Reply:
I would still keep it for a while, I have a feeling we will utilize also Prometheus in the near future.

Reply:
As discussed in person, for now, we will leave it and we will see as it evolves in the near future. Then we will refactor and remove not used code.